### PR TITLE
Escape `targetName` correctly in (hopefully) all cases

### PR DESCRIPTION
**Pull request by kevmoo**
Fixes https://github.com/GoogleCloudPlatform/functions-framework-dart/issues/83

### CODE REVIEW

File: functions_framework_builder/lib/src/utils.dart [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Borrowed from
+// https://github.com/google/json_serializable.dart/blob/4047521eac683886a014bb9ebe3fc19a2158c264/json_serializable/lib/src/utils.dart#L187
+
+/// Returns a quoted String literal for [value] that can be used in generated
```
Comment (kevmoo) [inline functions_framework_builder/lib/src/utils.dart:18]: This is ALSO code that should exist somewhere else. TBD...

File: functions_framework_builder/lib/src/utils.dart [hunk 2]
```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 Google LLC
```
Comment (grant) [inline functions_framework_builder/lib/src/utils.dart:1]: nit: 2020

See: 
https://github.com/google/new-project#source-code-headers

**Review by grant — APPROVED** [review]

File: functions_framework_builder/lib/src/utils.dart (hunk 2 above)
Comment (kevmoo) [reply functions_framework_builder/lib/src/utils.dart:1]: All of our other ones are `2021` – not sure why @subfuzion went this way...

Comment (kevmoo) [reply functions_framework_builder/lib/src/utils.dart:1]: Get ahead of the curve for next year?

**Pull request merged**